src/conf_mode/vpn_openconnect.py

```diff
@@ -63,17 +64,35 @@ def get_config():
 def verify(ocserv):
     if ocserv is None:
         return None
-
     # Check authentication
     if "authentication" in ocserv:
         if "mode" in ocserv["authentication"]:
             if "local" in ocserv["authentication"]["mode"]:
-                if not ocserv["authentication"]["local_users"] or not ocserv["authentication"]["local_users"]["username"]:
+                if "radius" in ocserv["authentication"]["mode"]:
+                    raise ConfigError('openconnect supports only one authentication mode. Currently configured \'local\' and \'radius\' modes')
```

Comment:
I think this message is slightly confusing and should emphasize support for only one mode _at the same time_. Maybe something like "OpenConnect authentication modes are mutually-exclusive, remove either local or radius from your configuration.

Reply:
Fixed